### DISCUSSION TITLE
How use backchannel

### DISCUSSION THREAD

**Discussion by BharadwajManne** [bluenviron/mediamtx#2889] [Questions]
### Question

How to use backchannel
 UNV Model IPC672LR-AX4DUPKC
This IP Camera supports backchannel and has an in-built camera. I can able to have a two-way talk when using it from NVR. Can you please let me know where am I lagging?

**Comment by aler9** [comment]
Hello, using back channels to send audio back to the camera is a feature that is currently not supported by MediaMTX. This feature is tracked here: #941

[a comment by an automated account is not shown]
